docs/source/specs/openapi.yml

```diff
@@ -442,7 +442,7 @@ paths:
             $ref: '#/definitions/Error'
       security:
       - api_key: []
-  /reports/ocp/cpu/:
+  /reports/ocp/inventory/cpu/:
```

Comment:
The Swagger paths don't match what's in `koku/api/urls.py`

Also, I'm not convinced we need the provider as part of the path. I don't know if we have plans to add collecting CPU & Memory stats out of AWS, but I think it's acceptable that the only handler for `/reports/inventory/cpu` right now is the OCP handler. It's not necessary to embed the provider into the URI.

Reply:
Good catch on the openapi.yml.   I think the rest of the discussion regarding paths can continue when/if we do have provider overlap.  I vote for just keeping the provider in the path for now so it's clear when integrating with the UI what provider the end point is intended for.